### PR TITLE
Automatically create containers that don't exist when adding with the…

### PR DESCRIPTION
**Pull request by lisdude**
… CSV editor.

### CODE REVIEW

**Comment by lisdude** [comment]
Without it, Storage.setAll will run before the promises from contextualIdentities.create resolve. So all of the new containers will get created but won't be in the CSV anymore. I just started learning JavaScript a couple of days ago so I wasn't quite sure how else to make it work.

I haven't noticed any performance issues but, honestly, I don't know. This is my first add-on experience too. 

On another branch I tried using 'babel-preset-env' and targeting Firefox >= 53, which has native support for everything the add-on is using (include async/await). I figured that would be safe since containers didn't appear until Firefox 50. That broke uglify, though, and I wasn't entirely sure how to fix that.

Basically I have no idea what I'm doing!
